Automattic/wp-calypso
# Refactor eCommerce plan features into a component

Related to Automattic/wc-calypso-bridge#1005

## Proposed Changes

* This PR refactors the plan features in the plans page for eCommerce trials into a reusable component.
  - We will be using the same structure/component in a new page for expired trials, but that work will be in a follow-up PR.

## Testing Instructions

* This one is a bit tricky to test, as nothing ought to be different from a functional or visual perspective.
* Using the production UI, enable Tracks logging, and then navigate to _Upgrades_ -> _Plans_ for a site on the free eCommerce trial
  - Click around the UI, including toggling the billing interval, expanding and contracting the feature sets, viewing the page in mobile viewports, and clicking on the upgrade CTAs
  - Make a note of the Tracks events that are fired
* Now repeat the steps above using a local dev build or the Calypso.live branch - you may need to add 
  - Verify that the behaviour remains the same
  - You should also test with and without the `plans/wooexpress-medium` feature flag enabled (e.g. `?flags=-plans/wooexpress-medium` to disable, and `?flags=plans/wooexpress-medium` to enable)

## Pre-merge Checklist

- [N/A] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [N/A] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?
